content/introduction/getting-started-with-sdks/index.mdx

```diff
@@ -0,0 +1,556 @@
+---
+
+import { Callout, Table, Steps, Tabs } from 'nextra/components';
+
+# Getting started with the JavaScript SDKs
+
+Sharetribe's
+[JavaScript Software Development Kit](https://sharetribe.github.io/flex-sdk-js/)
+(SDK) and
+[Integration SDK](https://sharetribe.github.io/flex-integration-sdk-js/)
+simplify interactions with our APIs. You can read more about features
+and functionalities in our
+[SDK GitHub Documentation](https://sharetribe.GitHub.io/flex-sdk-js/index.html).
+This guide covers
+
+- Installation
+- Initialization and instantiating the SDK
+- Creating a trusted SDK instance
+- Example use cases
+
+## Overview
+
+Our SDKs simplify making requests to the backend by handling
+authentication and providing convenient access to Sharetribe's APIs.
+They allow you to interact easily with
+
+- [Marketplace API](https://www.sharetribe.com/api-reference/marketplace.html),
+  for general interactions with marketplace data
+- [Integration API](https://www.sharetribe.com/api-reference/integration.html),
+  for advanced operational use
+- [Asset Delivery API](https://www.sharetribe.com/api-reference/asset-delivery-api.html),
+  for delivering assets configured in Console
+
+You can find more information on API usage and guidelines for choosing
+which to work with in
+[this article](https://www.sharetribe.com/docs/concepts/api-sdk/marketplace-api-integration-api/#when-to-use-the-marketplace-api).
+
+There are two JavaScript SDKs you can use to access the APIs
+
+<br />
+<Table>
+  <thead>
+    <Table.Tr>
+      <Table.Th />
+      <Table.Th>Sharetribe SDK</Table.Th>
+      <Table.Th>Sharetribe Integration SDK</Table.Th>
+    </Table.Tr>
+  </thead>
+  <tbody>
+    <Table.Tr>
+      <Table.Td>**Access level**</Table.Td>
+      <Table.Td>Limited to user-scope and public data</Table.Td>
+      <Table.Td>Full marketplace (operator-level) access</Table.Td>
+    </Table.Tr>
+    <Table.Tr>
+      <Table.Td>**API access**</Table.Td>
+      <Table.Td>Marketplace API and Asset Delivery API</Table.Td>
+      <Table.Td>Integration API</Table.Td>
+    </Table.Tr>
+    <Table.Tr>
+      <Table.Td>**Required Parameters**</Table.Td>
+      <Table.Td>
+        Client ID (Client secret for trusted contexts)
+      </Table.Td>
+      <Table.Td>Client ID, client secret</Table.Td>
+    </Table.Tr>
+    <Table.Tr>
+      <Table.Td>**Secure context needed**</Table.Td>
+      <Table.Td>When initialised with a trusted access token</Table.Td>
+      <Table.Td>Yes</Table.Td>
+    </Table.Tr>
+  </tbody>
+</Table>
+
+You'll notice we mention secure and trusted contexts in the above
+comparison. Some requests must be made from a _trusted context_, meaning
+the SDK must be initialized with both the client ID and client secret.
+As a result, all Integration SDK requests are performed within a trusted
+context.
+
+All trusted requests must be made from a _secure context_, meaning they
+need to be made from a server-side environment. An easy way to do this
```

Comment:
```suggestion
need to be made from a server-side environment. One way to do this
```

I would avoid using "easy". Someone less technical might be demotivated if we say that setting up a server is "easy". Easy for some for sure, not for everyone.